api/setup.cfg

```diff
@@ -40,6 +40,7 @@ DJANGO_SETTINGS_MODULE = mysagw.settings
 filterwarnings =
     error::DeprecationWarning
     error::PendingDeprecationWarning
+    ignore:LocalizedAutoSlug is deprecated and will be removed in the next major version\.:DeprecationWarning
```

Comment:
Sure we want to ignore this instead of fix/workaround, especially in a new project?

Reply:
This happens internally in `django-localized-fields` when `from localized_fields.fields import LocalizedCharField, LocalizedTextField`.